pkg/config/config.go

```diff
@@ -937,12 +940,18 @@ func (c *Config) Write() error {
 	return nil
 }
 
-// Reload reloads the configuration from containers.conf files
+// InvalidateCache invalidates the config cache.  This function is meant to be
+// used for long-running processes that need to reload potential changes made to
+// the cached containers.conf files.
+func InvalidateCache() {
+	configMutex.Lock()
+	defer configMutex.Unlock()
+	configErr = nil
+	config = nil
+}
+
+// Reload clean the cached config and reloads the configuration from containers.conf files
 func Reload() (*Config, error) {
```

Comment:
I don't feel very strongly about it, but we don't need `Reload()` anymore as `InvalidateCache` and `Default` can be called in sequence.

Reply:
I think Reload is clearer for the user,  I would be fine with making Invalidate as a private function,